packages/components/src/RichTextEditor/RichTextContent/_docs/RichTextContent.stories.tsx

```diff
@@ -22,3 +24,22 @@ export const Playground: Story = {
     chromatic: { disable: false },
   },
 }
+
+export const ReadOnly: Story = {
+  parameters: {
+    chromatic: { disable: false },
+  },
+  render: args => (
+    <>
+      <Label classNameOverride="block mb-6" id="sb--rich-text-content-label">
```

Comment:
I'm dubious on actually using the label component here in this example as a `content editable="false"` is akin to a regular div `RichTextContent` essentially is, what do folks thing about using a heading instead? CC: @dougmacknz  if you have any thoughts here on this?

Reply:
Yeah true label probably isn't correct, but it did make it really easy to match the styling once the real RTE comes in.

Heading doesn't really make sense IMO. Could just be a div I guess?

Reply:
Yeah that's fair. If that's the case then I should probably nix the aria-labelledby hey

Reply:
aria-labelledby pointing to a div is totally fine